### ISSUE TITLE
Latest Python buildpack does not support example app

### ISSUE THREAD

**Issue by peterellisjones** [cloudfoundry/cf-acceptance-tests#407]
CF acceptance tests python test app only support Python 2.7 (https://github.com/cloudfoundry/cf-acceptance-tests/blob/master/assets/python/runtime.txt) however the latest CF Python buildpack (1.7.7) deprecates Python 2.7 https://github.com/cloudfoundry/python-buildpack/releases/tag/v1.7.7

Example failing output:

```
   -----> Python Buildpack version 1.7.7
   -----> Supplying Python
          **ERROR** Could not install python: no match found for 2.7.x in [3.5.7 3.5.9 3.6.9 3.6.10 3.7.5 3.7.6 3.8.0 3.8.1]
   Failed to compile droplet: Failed to run all supply scripts: exit status 14
   Exit status 223
   Cell 877186a1-8148-43ff-b3bc-57c1590e9884 stopping instance 9b114df3-bf52-4e69-8bbd-2617fb4b8f14
   Cell 877186a1-8148-43ff-b3bc-57c1590e9884 destroying container for instance 9b114df3-bf52-4e69-8bbd-2617fb4b8f14
   Cell 877186a1-8148-43ff-b3bc-57c1590e9884 successfully destroyed container for instance 9b114df3-bf52-4e69-8bbd-2617fb4b8f14
FAILED
Error staging application: App staging failed in the buildpack compile phase
[AfterEach] Buildpacks
  /go/src/github.com/cloudfoundry/cf-acceptance-tests/detect/buildpacks.go:27

...

[Fail] [detect] Buildpacks python [It] makes the app reachable via its bound route 
/go/src/github.com/cloudfoundry/cf-acceptance-tests/detect/buildpacks.go:81

[Fail] [detect] Buildpacks python [It] makes the app reachable via its bound route 
/go/src/github.com/cloudfoundry/cf-acceptance-tests/detect/buildpacks.go:81

[Fail] [detect] Buildpacks python [It] makes the app reachable via its bound route 
/go/src/github.com/cloudfoundry/cf-acceptance-tests/detect/buildpacks.go:81

```

**Comment by davewalter** [comment]
Hi @peterellisjones,

Now that we have merged #408, we _think_ that this should be resolved. Would you mind verifying the changes on the `release-candidate` branch to confirm this, and close this out if so?

Thanks,
Dave

**Comment by vitreuz** [comment]
Hey @peterellisjones,

We hope this fix addresses the issue, and will go ahead and close the issue.
Thank you for reporting it. Please feel free to reopen this issue if the problem persists.

Thanks, Sebastian & @ericpromislow
